### DISCUSSION TITLE
Is there a way to color folders like in Gdrive ?

### DISCUSSION THREAD

**Discussion by Mysterius** [9001/copyparty#54] [Q&A]
Hello,

Everything is in the title ! Is there a way to chose a color for specific folders to make them stand out ?

**Comment by 9001** [comment]
Hey o/ and thx for trying out copyparty :grin: 

Sorry to say there's currently nothing like that right out of the box, but that doesn't necessarily have to stop us -- it might be possible to get something good enough just by using custom CSS.

Granted, there won't be a fancy menu to pick a color or anything like that, but maybe that's okay?

say you make a file named (for example) `custom.css` in the weboot (or somewhere else), with the following contents:

```css
a[href="/pub/demo/music/Sabrepulse%20-%20First%20Crush/"]::before { content: '⬤ '; color: #f0a }
```

...and then have copyparty load that into all the pages with the commandline argument `--css-browser=/custom.css` then it'll look something like this:

## in the default list-view:
![Screenshot_2023-10-17_20-27-03-or8](https://github.com/9001/copyparty/assets/241032/d18e36de-bb5f-48a9-8b25-63b669242bcc)

## ...and in the grid-view
![Screenshot_2023-10-17_20-28-34-or8](https://github.com/9001/copyparty/assets/241032/952e8589-07e5-4796-a3fb-acbb6b542c0a)

----

to add more folders, just copy that line, and change the path between the quotes -- the best way to get the right path is probably to navigate into the folder itself, and copy the full URL from your browser's address bar (`https://a.ocv.me/pub/demo/music/Sabrepulse%20-%20First%20Crush/`) and then cut away the stuff before the first `/` in there.

And if you give yourself both the `w`rite and `d`elete permissions for the volume that the css file is in, you'll be able to edit the css straight in the file browser -- so that makes it a bit less of a hassle to add folders that way I hope :)

Let me know what you think :grin: :+1:

**Comment by Mysterius** [comment]
Thank you for taking the time to write a detailed answer, I'll look into all that :)
